apps/mull-ui/src/app/components/chat-input/chat-input.scss

```
@@ -0,0 +1,45 @@
+@import '../../../variables.scss';
+
+.chat-input-container {
+  width: 100%;
+  position: fixed;
+  bottom: 4.75rem;
```

There is already a var for this. Make this `4.8rem`, which is the height of the bottom nav bar

```suggestion
  bottom: $bot-nav-bar-height-mobile;
```